contrib/coredns/policy/dnstap/policytap.proto

```diff
@@ -1,10 +1,11 @@
-syntax = "proto2";
+syntax = "proto3";
```

Comment:
please copy the proto from 
https://github.com/Infoblox-CTO/golang-dnstap/blob/Coredns-Support/dnstap.pb/policytap.proto
to make sure they are the same

Reply:
Fixed